bokehjs/src/coffee/common/hittest.coffee

```diff
@@ -35,7 +35,13 @@ class HitTestResult
       indices: []
     }
     # 2d for all for multilines and multi patches
-    @['2d'] = {indices: []}
+    @['2d'] = {
+      # index of the glyph in the multiglyph
+      indices: []
+      # object containing glyph_index: [point_index] of the nearest
+      # point to the crossed segment
+      point_indices: {}
+    }
```

Comment:
I think this is the only "controversial" part. There has to be an "indices"/array pair to support selections, but it'd be useful to know which points inside a line are selected. My solution was the `point_indices` object which contains the multi_line index and an array of line points.

I'm open to feedback on changing the implementation and/or naming.